src/settings/SettingPage/getCRUDActions.js

```diff
@@ -0,0 +1,106 @@
+import { buildUrl } from '@folio/stripes/smart-components';
+
+export const getCRUDActions = ({
+  finishedResourceName,
+  mutator: { [finishedResourceName]: resourceMutator },
+  location,
+  history,
+  match: { path },
+  onUpdateRecordError,
+  onDeleteSuccess,
+  onDeleteError,
+}) => {
+  const transitionToParams = params => {
+    const url = buildUrl(location, params);
+
+    history.push(url);
+  };
+
+  const createRecord = record => resourceMutator.POST(record)
+    .catch(error => {
+      onUpdateRecordError(record, error);
+
+      throw error;
+    });
+
+  const editRecord = record => resourceMutator.PUT(record)
+    .catch(error => {
+      onUpdateRecordError(record, error);
+
+      throw error;
+    });
+
+  // const showDeleteRecordSuccessfulMessage = record => {
+  //   const message = getDeleteRecordSuccessfulMessage(record);
+
+  //   calloutRef.current.sendCallout({ message });
+  // };
+
+  // const showDeleteRecordErrorMessage = (record, error) => {
+  //   if (withExceptionModal && error.status === 409) {
+  //     setShowExceptionModal(true);
+
+  //     return;
+  //   }
+
+  //   const message = getDeleteRecordErrorMessage(record);
+
+  //   calloutRef.current.sendCallout({
+  //     type: 'error',
+  //     message,
+  //   });
+  // };
+
+  const deleteRecord = async record => {
+    try {
+      await resourceMutator.DELETE(record);
+
+      transitionToParams({
+        _path: `${path}/view`,
+        layer: null,
+      });
+
+      onDeleteSuccess(record);
+    } catch (error) {
+      onDeleteError(record, error);
+    }
+  };
+
+  // const deleteMatchProfile = async record => {
```

Comment:
Will commented code to be changed/removed?

Reply:
Removed